test/utils/mocks/modules/paymentProcessor/PP_Streaming_v1AccessMock.sol

```diff
@@ -54,4 +54,12 @@ contract PP_Streaming_v1AccessMock is PP_Streaming_v1 {
     {
         return _validPaymentToken(_token);
     }
+
+    function original_getStreamingDetails(bytes16 flags, bytes32[] memory data)
```

Comment:
per our brand new repository code of conduct, these functions should be called exposed_XYZ(). We'll need to check the whole repo anyway, but maybe we can make the future intern's life a bit easier :)